packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -98,10 +98,6 @@ type HookType =
   | 'useImperativeHandle'
   | 'useDebugValue';
 
-// the first instance of a hook mismatch in a component,
-// represented by a portion of its stacktrace
-let currentHookMismatchInDev = null;
```

Comment:
we were including the trace so peeps could tell which (possibly custom hook) was the offending one. fine to remove?

Reply:
right, of course the warning would have its own trace. thanks!